msal/default_web_request_manager.go

```diff
@@ -109,13 +111,19 @@ func (wrm *defaultWebRequestManager) GetWsTrustResponse(ctx context.Context, aut
 		soapAction = SoapActionDefault
 	}
 
+	if endpoint.URL == "" {
```

Comment:
It know the default web request manager is leaving, but we should at least take the panic out